src/PhoneInputValidator.php

```diff
@@ -81,15 +81,14 @@ protected function validateValue($value)
      */
     public function clientValidateAttribute($model, $attribute, $view) {
 
-        $telInputId = Html::getInputId($model, $attribute);
         $options = Json::htmlEncode([
             'message' => \Yii::$app->getI18n()->format($this->message, [
                 'attribute' => $model->getAttributeLabel($attribute)
             ], \Yii::$app->language)
         ]);
 
         return <<<JS
-var options = $options, telInput = $("#$telInputId");
+var options = $options, telInput = $(attribute.input);;
```

Comment:
@mzglinski where does this `attribute.input` come from?

Reply:
From the anonymous function which wraps this code. 
https://www.yiiframework.com/doc/api/2.0/yii-validators-validator#clientValidateAttribute()-detail

Reply:
@mzglinski right, you're right. Thanks!